### PR TITLE
docs: one year of PRQL

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

File: website/content/posts/2023-02-02-one-year/_index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+toc: true
+---
+
+A year ago,
+[a proposal for a modern SQL replacement](https://news.ycombinator.com/item?id=30060784)
+was posted to Hacker News. It immediately sparked the interest of many people
+dealing with data. The project grew a community of people who are now developing
+the language, tooling, and the idea of a modern relational language. Since then
+we've opened 1577 issues & PRs on [our main repo](https://github.com/PRQL/prql),
+submitted 4211 comments, and made 1176 commits.
```
Comment (max-sixty) [inline website/content/posts/2023-02-02-one-year/_index.md:14]: Cool stats!

File: website/content/posts/2023-02-02-one-year/_index.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+---
+title: One year of PRQL
+date: 2023-02-02
+layout: article
+toc: true
+---
+
+A year ago,
+[a proposal for a modern SQL replacement](https://news.ycombinator.com/item?id=30060784)
+was posted to Hacker News. It immediately sparked the interest of many people
```
Comment (max-sixty) [inline website/content/posts/2023-02-02-one-year/_index.md]:
```suggestion
A year ago, we posted
[a proposal for a modern SQL replacement](https://news.ycombinator.com/item?id=30060784)
to Hacker News. It immediately sparked the interest of many people
```

File: website/content/posts/2023-02-02-one-year/_index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+- small quality-of-live language features (e.g. syntax for
+  [f-strings, dates, coalesce operator](https://prql-lang.org/book/syntax.html),
+  [switch](https://github.com/PRQL/prql/issues/504)),
+
+- design of basic [transforms](https://prql-lang.org/book/transforms/index.html)
+  and their [interactions](https://github.com/PRQL/prql/issues/300),
+
+- fundamentals of how [functions](https://github.com/PRQL/prql/issues/444) and
+  pipelines are [evaluated](#define-functional-semantics).
```
Comment (max-sixty) [inline website/content/posts/2023-02-02-one-year/_index.md:30]: Very nice list

File: website/content/posts/2023-02-02-one-year/_index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+)
+
+from invoices
+take_oldest 3
+```
+
+> Function `take_oldest` is used in the main pipeline, just any other regular
+> transform.
+
+### Relational Query
```
Comment (max-sixty) [inline website/content/posts/2023-02-02-one-year/_index.md:115]: Great to include these two items!

**Review by max-sixty — COMMENTED** [review]
It's better than I thought we get to!

Should we add something on "the way forward" or "what we're struggling with" or some call-to-action to contribute? Or maybe this is nicely self-contained?

File: website/content/posts/2023-02-02-one-year/_index.md (hunk 3 above)
Comment (richb-hanover) [reply website/content/posts/2023-02-02-one-year/_index.md:30]: I might switch the order:
* Design of basic transforms
* Fundamentals of functions/pipelines
* small quality of **life** features (not "live")

File: website/content/posts/2023-02-02-one-year/_index.md (hunk 4 above)
Comment (richb-hanover) [reply website/content/posts/2023-02-02-one-year/_index.md:115]: For each testimonial, you could include:

My sentiment after the Pretty Reports line: 

> I can no longer bring myself to write bare SQL - PRQL makes building queries so easy. 

And maybe this for the CLI...

> ... I can use `prql-query` to execute PRQL queries against database engines.

**Comment by richb-hanover** [comment]
This is a great one-year retrospective!

File: website/content/posts/2023-02-02-one-year/_index.md (hunk 4 above)
Comment (aljazerzen) [reply website/content/posts/2023-02-02-one-year/_index.md:115]: It seems too promotional to quote ourselves.

**Comment by aljazerzen** [comment]
> Should we add something on "the way forward" or "what we're struggling with" or some call-to-action to contribute? Or maybe this is nicely self-contained?

I wanted to add a call for contributors at the bottom, but there already the footer doing the same thing.

The way forward it the roadmap.

**Pull request merged**